WordPress/wporg-documentation-2022
# Update local header layout

Closes WordPress/wporg-main-2022#66 

Depends on ~https://github.com/WordPress/wporg-parent-2021/pull/103~, ~https://github.com/WordPress/wporg-mu-plugins/pull/443~  and https://github.com/WordPress/wporg-mu-plugins/pull/439

- Changes background and border colors
- Adds site title 'Documentation' on pages other than home

Home header background color can be updated in content.

## Screenshots (outdated, see [below](https://github.com/WordPress/wporg-documentation-2022/pull/67#issuecomment-1714789899))

### Home

| Desktop | Mobile |
|--------|-------|
| ![localhost_8888_(Desktop) (2)](https://github.com/WordPress/wporg-documentation-2022/assets/1017872/f84f747f-ca07-470b-ac0a-713920f9ba9e) | ![localhost_8888_(Samsung Galaxy S20 Ultra) (1)](https://github.com/WordPress/wporg-documentation-2022/assets/1017872/7fbeb919-0905-4c2d-9d79-38cb4927429c) |

### Topic landing page

| Desktop | Mobile |
|--------|-------|
| ![localhost_8888_technical-guides_(Desktop)](https://github.com/WordPress/wporg-documentation-2022/assets/1017872/b7c9fb06-65b7-4e9f-afff-b06d8226c640) | ![localhost_8888_technical-guides_(Samsung Galaxy S20 Ultra)](https://github.com/WordPress/wporg-documentation-2022/assets/1017872/838fa070-a6e1-4c2a-89ff-27228da236eb) |

### Article

| Desktop | Mobile |
|--------|-------|
| ![localhost_8888_article_block-directory_(Desktop)](https://github.com/WordPress/wporg-documentation-2022/assets/1017872/791b0057-172e-445e-aa4e-8a6586f6a33d) | ![localhost_8888_article_block-directory_(Samsung Galaxy S20 Ultra)](https://github.com/WordPress/wporg-documentation-2022/assets/1017872/2a115164-4b69-4bf9-844a-153c907bde14) |

### How to test the changes in this Pull Request:

1. Ensure your mu-plugins and parent theme are updated ~on the branches detailed above~
2. Check the homepage, topic landing pages and a single article page
